ios/AppDelegate.swift

```diff
@@ -11,15 +11,29 @@ import React_RCTAppDelegate
 import ReactAppDependencyProvider
 import ExpoModulesCore
 import Firebase
-
+import Expo
  
 @main
 class AppDelegate: ExpoAppDelegate, UNUserNotificationCenterDelegate {
+  var window: UIWindow?
+  var reactNativeDelegate: ExpoReactNativeFactoryDelegate?
+  var reactNativeFactory: RCTReactNativeFactory?
+  
   override func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil) -> Bool {
-    self.moduleName = "NewExpensify"
-    self.dependencyProvider = RCTAppDependencyProvider()
-    self.initialProps = [:]
+    let delegate = ReactNativeDelegate()
+    let factory = ExpoReactNativeFactory(delegate: delegate)
```

Comment:
Why do we need Expo's factory here? Does this not work?
```swift
let factory = RCTReactNativeFactory(delegate: delegate)
```

Reply:
bumping on this one @war-in

Reply:
@adhorodyski Done! https://github.com/Expensify/App/pull/60421/commits/d4608ff827fb8eb090cff943a01794730b04352d